utils/lit/lit/TestingConfig.py

```diff
@@ -32,7 +61,12 @@ def fromdefaults(litConfig):
                                      [os.environ.get('PATH','')])
 
         all_path = strip_dxil_validator_path(all_path)
-
+        if sys.platform == 'win32':
+            required_tools = [
+                'cmp.exe', 'grep.exe', 'sed.exe', 'diff.exe', 'echo.exe', 'ls.exe']
+            path = _find_git_windows_unix_tools(required_tools)
+            if path:
```

Comment:
I think this check is unnecessary now since `_find_git_windows_unix_tools` cannot fail (will raise an exception if the path cannot be found).

Reply:
Done.